js/constants.js

```diff
@@ -1,12 +1,12 @@
 /*!
- * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
 
 module.exports = {
-  CONTEXT_FILENAME: 'example-v1.jsonld',
-  CONTEXT_URL_V1: 'https://w3id.org/example/v1',
+  CONTEXT_FILENAME: 'vc-refresh-2021-v1.jsonld',
+  CONTEXT_URL_V1: 'https://w3id.org/vc-refresh-2021/v1',
   // value between 0x0 and 0x7FFF for globally registered term codec values
   // or >= 0x8000 for app-specific local terms
-  CBORLD_VALUE: 0x0
+  CBORLD_VALUE: 0x21
```

Comment:
Is this required? If yes, then it needs to be also added on the list. https://github.com/digitalbazaar/cborld/blob/main/lib/codecs/registeredContexts.js